lib/elixir/lib/io/ansi.ex

```diff
@@ -170,6 +170,41 @@ defmodule IO.ANSI do
   @doc "Sends cursor home."
   defsequence(:home, "", "H")
 
+  @doc """
+  Sends cursor to the absolute position specified by `lines`, `columns`, 
+  with 0,0 being the top left corner.
+  """
+  @spec cursor(integer, integer) :: String.t()
+  def cursor(line, column) when line >= 0 and column >= 0, do: "\e[#{line};#{column}H"
+
+  @doc "Sends cursor `lines` up (defaults to 1)."
+  @spec cursor_up :: String.t()
+  def cursor_up(), do: "\e[A"
+
+  @spec cursor_up(integer) :: String.t()
+  def cursor_up(lines) when lines > 0, do: "\e[#{lines}A"
```

Comment:
Last thing, promise 😅 

The docs are now assigned to `cursor_up/0`, leaving `cursor_up/1` without any documentation.

Reply:
Actually thanks for all this feedback, as you can tell it's been a while and my elixir is a little rusty :)

is there a normal way to solve this problem?

Reply:
Multi-arity functions usually behave somehow different, so their documentation is usually not the same. There are some example in the Elixir codebase, [`Path.absname/1`](https://hexdocs.pm/elixir/Path.html#absname/1) and [`Path.absname/2`](https://hexdocs.pm/elixir/Path.html#absname/2) could be one of them.